### DISCUSSION TITLE
WAX Wallet Integration

### DISCUSSION THREAD

**Discussion by shaffy9633** [WalletConnect/walletconnect-monorepo#2589] [Developer Support]
Hi I want to request or possibly ask if you guys would be integrate WAX Cloud Wallet any time soon? Is there any thing we could do on our end to do this. i.e. fill up a form etc. We are currently using WallectConnect for our website https://www.blumint.com/. We are a web3 gaming platform, and alot of the games that are coming on the platform are using WAX wallets.  Hoping that someone could respond to this ASAP.

**Comment by finessevanes** [comment]
Hey - if you want WAX wallets to work with WalletConnect, then they need to implement our SDKs.

**Reply by shaffy9633** [reply]
noted @finessevanes thanks for responding. Will connect with their team and get this done ASAP. thank you! :)
